model/metrics/database-metrics.yaml

```diff
@@ -33,6 +33,154 @@ groups:
       - ref: db.namespace
         requirement_level:
           conditionally_required: If available.
+      - ref: thread.id
+        requirement_level:
+          conditionally_required: If available
+
+  - id: metric.db.client.operation.wait_time
+    type: metric
+    metric_name: db.client.operation.wait_time
+    brief: "Time spent waiting by database client operations, by category (if available)."
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    extends: attributes.db.client.minimal
+    attributes:
+      - ref: db.collection.name
+        requirement_level:
+          conditionally_required: >
+            If readily available. The collection name MAY be parsed from the query text,
+            in which case it SHOULD be the first collection name in the query.
+      - ref: db.system
+        # TODO: Not adding to the minimal because of https://github.com/open-telemetry/build-tools/issues/192
+        requirement_level: required
+      - ref: network.peer.address
+        brief: Peer address of the database node where the operation was performed.
+        requirement_level:
+          recommended: If applicable for this database system.
+        note: >
+          Semantic conventions for individual database systems SHOULD document whether `network.peer.*` attributes are applicable.
+          Network peer address and port are useful when the application interacts with individual database nodes directly.
+
+          If a database operation involved multiple network calls (for example retries), the address of the last contacted node SHOULD be used.
+      - ref: network.peer.port
+        requirement_level:
+          recommended: If and only if `network.peer.address` is set.
+      - ref: db.namespace
+        requirement_level:
+          conditionally_required: If available.
+      - ref: thread.id
```

Comment:
Nit: maybe say "if available and stable for the lifetime of the operation".

IF using, e.g. async IO and the handler of the response is different than the calling thread - you don't want to create new timeseries because thread.id changes.  Can we make that more explicit?

Reply:
Good call, done